Implemented anti-vorime'd technology

ellomenop opened the pull request:
Makes first roll of first boon reward offer all 4 core boons

Commit:
Implemented anti-vorime'd technology

Makes first roll of first boon reward offer all 4 core boons

Commit:
Added LootChoiceExt dependency

Credit to Magic Gonads

Commit:
Minor documentation fixes

Commit:
Adding ModConfigMenu support

parasHadesMods commented on DontGetVorimed/DontGetVorimed.lua:
```diff
@@ -0,0 +1,42 @@
+-- When the first room is created, set the number of loot choices to 4
+ModUtil.WrapBaseFunction("ChooseStartingRoom", function ( baseFunc, currentRun, roomSetName )
+  if config.Enabled then
+    LootChoiceExt.Choices = 4
```
What if the first room offers a hammer?

ellomenop replied:
it doesn't matter, this just sets the config on first room as a convenient hook.  If we have a consistent "run is starting" hook I can change it.

Basically this config will persist until its explicitly reset

Commit:
Removed ModConfigMenu integration

Removed ModConfigmenu integration because config will be managed in another system
Clarified documentation
